default/stringtables/en.txt

```diff
@@ -1102,6 +1102,30 @@ Starlanes Everywhere
 RULE_STARLANES_EVERYWHERE_DESC
 Starlanes are generated between every pair of systems, regardless of geometry. Lanes may still be removed during a game, however. Lanes are also not rendered on the galaxy map.
 
+RULE_HABITABLE_SIZE_TINY
+Tiny planets Habitable size
```

Comment:
This names are too long to fit in the rules list, but that can be fixed later...